tests/yamitranscode.cpp

```diff
@@ -109,7 +109,7 @@ static bool processCmdLine(int argc, char *argv[], TranscodeParams& para)
         return false;
     }
 
-    while ((opt = getopt_long_only(argc, argv, "W:H:b:f:c:s:i:o:N:h:t:", long_opts,&option_index)) != -1)
+    while ((opt = getopt_long_only(argc, argv, "W:H:b:f:c:s:i:o:N:h:t:L", long_opts,&option_index)) != -1)
```

Comment:
some comments in my side:
1.it is better to print out which frame got the max latecny?
2. is it OK to add the log option to print the latency per frame into the log file?